### PR TITLE
added data generation script

### PR DESCRIPTION
**Pull request by enague**

### CODE REVIEW

File: database/data/faker_data.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+
+//listing_names
+
+// const csvStream_names = csv.createWriteStream({headers: false})
+// const ws_names = fs.createWriteStream('faker_listing_names.csv')
+
+// csvStream_names.pipe(ws_names);
+// for(let i = 0; i < 10000001; i+=1) {
+// 	const obj = {}
```
Comment (kiannaquach) [inline database/data/faker_data.js]: don't forget your semi-colons :)

File: database/data/faker_data.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+// 		obj['additionalcomments'] = faker.lorem.word()
+// 	csvStream_amenities.write(obj);
+// }
+
+
+// //tblOverview
+
+//ALTERNATIVE
+// let tblOverview = []
+// const headers = "0,ab,sit,Quia dolor numquam quia corporis quisquam delectus.,Rem et sed.,Deserunt delectus sed voluptatum debitis ipsam consectetur eum dolor magni.,Dolorem fugiat ut dolorem qui error et suscipit nisi.,Natus et omnis eum ipsam culpa est molestias.,Quisquam quis dolor tenetur consequatur rem suscipit.,Assumenda libero voluptatibus esse voluptates quia id.,4,2,2,3\n"
```
Comment (kiannaquach) [inline database/data/faker_data.js]: why are your headers lorem text?

File: database/data/faker_data.js [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+// //theSpace
+
+// let theSpace = []
+// fs.writeFileSync('faker_theSpace.csv', "0, thespace\n", 'utf8');
+
+// for(let i = 1; i < 10000001; i+=1) {
+// 	const thespace = faker.lorem.paragraph()
+// 	theSpace.push([i,thespace])
+
+// 	if(i % 50000 === 0) {
```
Comment (kiannaquach) [inline database/data/faker_data.js]: have you tried chunking for every 100,000?

File: database/data/faker_data.js [hunk 4]
```diff
@@ -0,0 +1,133 @@
+const faker = require('faker');
```
Comment (kiannaquach) [inline database/data/faker_data.js]: don't forget to lint your code 👍

File: client/src/App.jsx [hunk 5]
```diff
@@ -51,27 +51,28 @@ class App extends React.Component {
...
-        const s3 = results.data[0].interactionWithGuests.split('\n\n');
-        const s4 = results.data[0].otherThingsToNote.split('\n\n');
-        const br = results.data[0].noOfBedrooms;
-        const bd = results.data[0].noOfBeds;
-        const ba = results.data[0].noOfBaths;
-        const gs = results.data[0].noOfGuests;
-        const hh1 = results.data[0].homeHighlights1;
-        const hh2 = results.data[0].homeHighlights2;
-        const hh3 = results.data[0].homeHighlights3;
+        console.log(results.data.rows[0])
```
Comment (Jshhhh) [inline client/src/App.jsx:54]: You should take out console logs

**Review by Jshhhh — COMMENTED** [review]
Good work overall. Consider PRing to master as preparing for production. So I think you should remove console logs

File: client/src/App.jsx (hunk 5 above)
Comment (enague) [reply client/src/App.jsx:54]: alright will do!

**Pull request merged**